Engine_Revit_UI/Convert/Architecture/FromRevit/Ceilings.cs

```diff
@@ -77,7 +77,10 @@ public static List<oM.Architecture.Elements.Ceiling> CeilingsFromRevit(this Ceil
 
                 newCeiling.UpdateValues(settings, ceiling);
 
-                newCeiling.CustomData.Add("CeilingPattern", ceiling.CeilingPattern(settings, planarSurface));
+                newCeiling.CustomData.Add(BH.Engine.Adapters.Revit.Convert.CeilingPattern, ceiling.CeilingPattern(settings, planarSurface));
+                if (disctionary.Values.Sum(x => x.Count) != 0)
```

Comment:
For some reason this can be null, so the statement needs to be rephrased to:
`if (disctionary.Values.Where(x => x != null).Sum(x => x.Count) != 0)`